google-drive/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id("ivy.feature")
+    kotlin("kapt")
+}
+
+android {
+    namespace = "com.ivy.googledrive"
+}
+
+dependencies {
+    implementation(projects.ivyBase)
+    implementation(projects.ivyResources)
+    implementation(projects.tempLegacyCode)
+    api(libs.google.services.drive) {
```

Comment:
"api" is banned because it leads to complexity and slow build times, use `implementation`